draft-ietf-httpbis-digest-headers.md

```diff
@@ -407,7 +411,7 @@ Registrations MUST include the following fields:
      - e.g. according to the type and status of the primary document
      in which the algorithm is defined;
      "insecure" when the algorithm is insecure;
-     "reserved" when Digest algorithm references a reserved token value
+     "reserved" when the digest algorithm references a reserved token value
```

Comment:
```suggestion
     "reserved" when the algorithm references a reserved token value
```

this would make it consistent with the line above

Reply:
rebased and merged. Thx